README.md

```diff
@@ -73,7 +73,7 @@ _(There are also other ways to perform a similar task, including using `curl -is
 
 * `OHHTTPStubs` is compatible with **iOS5+**, **OS X 10.7+**, **tvOS**.
 * `OHHTTPStubs` also works with `NSURLSession` as well as any network library wrapping them.
-* `OHHTTPStubs` is **fully compatible with Swift 2.2, 2.3, 3.0 and 3.1**.
+* `OHHTTPStubs` is **fully compatible with Swift 2.2, 2.3, 3.0, 3.1 and 4.2**.
```

Comment:
Maybe at this point we'd just state "from 2.2 to 4.2"… although I'm not sure that's really true. We might have dropped 2.x support at some point and forgot about updating that part of the README maybe? Would be good to check if we're still compatible with 2.x… or remove that from the sentence altogether as probably nobody uses 2.2 anymore…